ports/rp2/Makefile

```diff
@@ -28,7 +30,9 @@ else
 BUILD ?= build-$(BOARD)
 endif
 
-$(VERBOSE)MAKESILENT = -s
```

Comment:
This PR no longer passes `-s` to the top-level make invocation, but this appears not necessary as the generated Makefile sets `.SILENT` at the top level based on the `VERBOSE` variable. I couldn't see a difference with or without `-s` at the top level.